src/synthesize.jl

```diff
@@ -65,12 +66,16 @@ function synthesize(atm, linelist, λs::AbstractVector{F}; metallicity::F=0.0, v
         number_densities = molecular_equilibrium(MEQs, layer.temp, layer.number_density,
                                                  layer.electron_density)
 
-        α[i, :] = line_absorption(linelist, λs, layer.temp, number_densities, atomic_masses, 
-                                  partition_funcs, ionization_energies, vmic*1e5)
+        α_cntm = LinearInterpolation(cntmλs,
```

Comment:
It might be useful to add a comment explaining that why we are allowing for extrapolation at all (i.e. to serve as as a heuristic for determining the window size of the lines). When I first saw this, I got very concerned (before I realized what was going on).

Reply:
I've added a comment to explain the interpolation, and I've tweaked the code to make extrapolation unnecessary.